### ISSUE TITLE
WCAG 2.2 本体 : 用語「アクセシビリティ サポーテッド (accessibility supported)」の注記 3

### ISSUE THREAD

**Issue by caztcha** [waic/wcag22#2173]
原文
> When a web technology is used in a way that is "accessibility supported," it does not imply that the entire technology or all uses of the technology are supported. Most technologies, including HTML, lack support for at least one feature or use. Pages conform to WCAG only if the uses of the technology that are accessibility supported can be relied upon to meet WCAG requirements.

現行訳
> あるウェブ技術が「アクセシビリティ サポーテッド」な方法で使用されている場合、その技術全体、又はその技術の全ての使用方法がサポートされていることを暗に示すものではない。HTML を含むほとんどの技術は、少なくとも一つの機能又は使用方法のサポートを欠いている。ページが WCAG に適合するのは、アクセシビリティ サポーテッドであるその技術の使用に依存することで WCAG の要件を満たすことことができる場合だけである。

修正案

> あるウェブ技術が「アクセシビリティ サポーテッド」な方法で使用されていても、その技術全体、又はその技術の全ての使用方法がサポートされていることを暗に示すわけではない。HTML を含むほとんどの技術は、少なくとも一つの機能又は使用方法のサポートを欠いている。その技術のアクセシビリティ サポーテッドである使用方法に依存することで WCAG の要件を満たすことができる場合だけ、ページは WCAG に適合する。
